air/src/options.rs

```diff
@@ -129,6 +129,21 @@ impl ProvingOptions {
         self
     }
 
+    /// Sets partitions for this [ProvingOptions].
+    ///
+    /// Partitions can be provided to split traces during proving and distribute work across
+    /// multiple devices. The number of partitions should be equal to the number of devices.
+    pub const fn with_partitions(mut self, num_partitions: usize) -> Self {
+        let hash_rate = match self.hash_fn {
+            HashFunction::Blake3_192 => 6,
```

Comment:
I think this could be $8$ as well. For this variant we just truncate the returned hash, but the function can still absorb up to 64 bytes in a single permutation.

Reply:
Changed to 8. The `match` is not really necessary but I kept it so that if  `HashFunction`s enum ever changes, this code will be revisited.